### PR TITLE
first draft of lockbox partner flow spec

### PR DESCRIPTION
**Pull request by misternu**
Open a PR and fill out the template later!

## Changelog
- What did you do (high level bullet points)?

## Link to issue:  
#152 

## Steps for QA/Special Notes:
- N/A

## Relevant Screenshots: 
- super important for UI tasks!

## Are you ready for review?:

- [ ] Added relevant tests
- [ ] Linked PR to the issue
- [ ] Added notes for QA/special notes
- [ ] Added revelant screenshots

### CODE REVIEW

File: spec/system/lockbox_partner_initialization_flow_spec.rb [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+  let!(:user) { FactoryBot.create(:user) }
+
+  it 'flows' do
+    # MAC coordinator makes a new lockbox partner
+    in_session "MAC Coordinator" do
+      login_as(user, :scope => :user)
+      visit "/"
+      click_on "Create new lockbox partner"
+
+      fill_in "Name", with: "#{Faker::Cannabis.strain.titleize} Clinic"
```
Comment (BintLopez) [inline spec/system/lockbox_partner_initialization_flow_spec.rb]: Lol

File: spec/system/lockbox_partner_initialization_flow_spec.rb [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+      fill_in "Street address", with: Faker::Address.street_address
+      fill_in "City", with: "Chicago"
+      fill_in "State", with: "IL"
+      fill_in "Zip code", with: "60606"
+      fill_in "Phone number", with: Faker::PhoneNumber.phone_number
+      click_on "Add New Partner"
+      expect(LockboxPartner.count).to eq 1
+    end
+
+    # TODO specify how the coordinator is informed that the lockbox parner has been created
```
Comment (BintLopez) [inline spec/system/lockbox_partner_initialization_flow_spec.rb]: For this I think asserting the path and/or messaging would suffice

File: spec/system/lockbox_partner_initialization_flow_spec.rb [hunk 3]
```diff
@@ -0,0 +1,96 @@
+require 'rails_helper'
```
Comment (scottyschup) [inline spec/system/lockbox_partner_initialization_flow_spec.rb]: I added some similar tests in #287 that can probably be consolidated with these
https://github.com/MidwestAccessCoalition/lockbox_rails/pull/287/files#diff-dc35c0d6c26c8e8f37d12016bb658c88

**Comment by misternu** [comment]
This is stale, closing

**Pull request closed without merging**